searchlib/src/vespa/searchlib/diskindex/disktermblueprint.cpp

```diff
@@ -38,12 +38,12 @@ getName(uint32_t indexId)
 
 }
 
-DiskTermBlueprint::DiskTermBlueprint(const FieldSpec & field,
+DiskTermBlueprint::DiskTermBlueprint(const FieldSpec field,
```

Comment:
Should this be un-`const`ed now that it's by value and is being `std::move`d away from?

Reply:
Yes, fixed